drivers/lcd/ssd1351.c

```diff
@@ -105,9 +105,7 @@
 
 /* Macro Helpers ************************************************************/
 
-#define SSD1351_MAX(a, b)        ((a) > (b) ? (a) : (b))
-#define SSD1351_MIN(a, b)        ((a) < (b) ? (a) : (b))
-#define SSD1351_CLAMP(n, a, b)   SSD1351_MIN(SSD1351_MAX(n, a), b)
+#define SSD1351_CLAMP(n, a, b)   MIN(MAX(n, a), b)
```

Comment:
Need to include `sys/param.h` in this file.

Reply:
Done.